.github/ISSUE_TEMPLATE/library-bug.yml

```diff
@@ -0,0 +1,36 @@
+name: Library Bug Report (developers)
+description: File a bug report on one of our NPM libraries
+title: "[Bug]: "
+labels: ["bug", "libraries", "Needs Triage"]
```

Comment:
Ah one more thing, I think the `Needs Triage` label is not currently applied because it does not exist with that name, the one which exists is called `triage`.